### PR TITLE
Wording to discourage note reporting

### PR DESCRIPTION
**Pull request by harry-wood**
Add some wording wrapped around the 'Report this note' link to explain to users to resolve the note instead.

Also re-arrange to place this down at the bottom with less prominence than the comment/resolve form.

Tackling the issue reported by DWG, that too many note reports are coming to them, where the users could deal with it themselves: https://github.com/openstreetmap/openstreetmap-website/issues/3071

notes-reporting-text2

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2883,6 +2883,7 @@ en:
         reactivate: Reactivate
         comment_and_resolve: Comment & Resolve
         comment: Comment
+        report_link_html: "If this note contains sensitive information that needs to be removed by an admin, you can %{link}. For all other cases, please resolve the note yourself with a comment"
```
Comment (pnorman) [inline config/locales/en.yml:2886]: This should contain a trailing period. Additionally, I'd remove "by an admin" as it doesn't matter to the user who removes it.

**Review by pnorman — CHANGES_REQUESTED** [review]

**Comment by AntonKhorev** [comment]
Sometimes users add a note, then resolve it (without making any changes to the map), then reopen, then report.

Probably there should also be some discouragement for resolving, so users don't close the notes *because* the information in notes is correct.

**Comment by harry-wood** [comment]
Yeah I guess users incorrectly "Resolve" due to all sorts of misunderstandings. Maybe the meaning of "Resolve" can be made clearer somehow. Should we have different buttons for different outcomes? Should we be allowed to resolve without commenting? Not sure. I'm not going to tackle that in this PR though.

"For all other cases" is the wording @woodpeck was suggesting (https://github.com/openstreetmap/openstreetmap-website/issues/3071). I was basically questioning this when I mentioned spam. Isn't it better to _report_ spam rather than _resolve_ it? Answer: no! At least not from the point of view of people dealing with too many reports! The wording is deliberately discouraging reporting quite strongly.

**Comment by AntonKhorev** [comment]
Resolving without commenting makes sense when you resolve your own note after you made the changes to the map.

"For all other cases" wording in #3581 was for "1. ... if they encounter a useless note" situation, not for every note someone might find.

**Comment by AntonKhorev** [comment]
What I just wrote basically says replace "For all other cases" with "If the note is useless", but maybe even that is not enough. This would require the user to know which notes are useless, and the kind of a user who'd report the note when they don't need to probably doesn't know that.

**Comment by harry-wood** [comment]
hmm. How about instead of "For all other cases..." we could say "For all other problems..." ?

**Comment by AntonKhorev** [comment]
Users have to know that there are *problems with the map* and *problems with the notes*. In case of *problems with the map* they should keep the notes about these problems open. In case of *problems with the notes* they should resolve the notes.

**Comment by AntonKhorev** [comment]
"For all other problems..." -> "For all other problems *with the note*..." is a better improvement.

**Comment by gravitystorm** [comment]
Thanks for the PR. I've rebased it, and reworked the styling to use bootstrap spacer utilities rather than custom CSS. And I've put the text in a `<small>` tag too, since it's not normally required reading for dealing with the note.

I hope this reduces the number of reports that DWG needs to deal with!

**Comment by harry-wood** [comment]
Thanks Andy! One `<small>` step @woodpeck !

**Comment by woodpeck** [comment]
Just wanted to say thank you, I think it is already having an effect!

**Comment by harry-wood** [comment]
Excellent!
I was just working on a follow-up tweak https://github.com/openstreetmap/openstreetmap-website/pull/3676 (Something else to help people realise they don't need to report)

**Pull request merged**